homeassistant/components/here_travel_time/sensor.py

```diff
@@ -172,7 +173,12 @@
         name, origin, destination, origin_entity_id, destination_entity_id, here_data
     )
 
-    async_add_entities([sensor], True)
+    def delayed_sensor_update(event):
```

Comment:
Instead create an entity method or function, inside the entity, that calls `self.async_schedule_update_ha_state(True)`.

Register this function as the callback to the start event listener.